content/openrouteservice.md

```diff
@@ -1,9 +1,442 @@
+Dort finden wir unter anderem historische PBF-Dateien, die wir für unsere Zwecke verwenden können.
+
+Um zu der Übersicht mit den historischen PBF-Dateien zu gelangen, klicken wir auf den Link "raw directory index":
+![Historische PBF-Dateien](../img/geofabrik_historic_data_overview.png)
+
+Dort wählen wir die älteste verfügbare PBF-Datei vom 02.01.2014 aus: `muenster-regbez-140101.osm.pbf`.
+![Historische PBF-Datei](../img/geofabrik_historic_data_selection.png)
+
+Nun laden wir die historische PBF-Datei herunter:
+
+```bash
+# Ordnerstruktur für historische und aktuelle Daten erstellen
+mkdir -p ors-docker-historic/files ors-docker-latest/files
+# Aktuelle Münster PBF-Datei herunterladen
+curl -C - https://download.geofabrik.de/europe/germany/nordrhein-westfalen/muenster-regbez-latest.osm.pbf -o ors-docker-latest/files/muenster-regbez-latest.osm.pbf
+# Historische Münster PBF-Datei herunterladen
+curl -C - https://download.geofabrik.de/europe/germany/nordrhein-westfalen/muenster-regbez-140101.osm.pbf -o ors-docker-historic/files/muenster-regbez-140101.osm.pbf
+```
+
+#### Zwei Openrouteservice-Container mit unterschiedlichen PBF-Dateien konfigurieren
+
+Wir erstellen zwei Konfigurationsdateien, die jeweils ein Profile `driving-car` aktivieren, aber unterschiedliche PBF-Dateien verwenden.
+
+```bash
+# Konfiguration mit aktuellem Münster PBF-File
+echo "ors.engine.profiles.driving-car.build.source_file=/home/ors/files/muenster-regbez-latest.osm.pbf" > ors_car_muenster_latest.env
+echo "ors.engine.profiles.driving-car.enabled=true" >> ors_car_muenster_latest.env
+# Konfiguration mit historischem Münster PBF-File
+echo "ors.engine.profiles.driving-car.build.source_file=/home/ors/files/muenster-regbez-140101.osm.pbf" > ors_car_historic_muenster.env
+echo "ors.engine.profiles.driving-car.enabled=true" >> ors_car_historic_muenster.env
+```
+
+#### Zwei Openrouteservice-Container mit unterschiedlichen PBF-Dateien starten
+
+Wir starten zwei Openrouteservice-Container, die jeweils auf Port 8080 (latest osm) und 8081 (historic osm) erreichbar sind und jeweils eine der Konfigurationsdateien verwenden.
+
+```bash
+# Alte Container entfernen
+docker rm -f ors-app || true
+docker rm -f ors-app-historic || true
+
+# Neuen Container mit aktuellem Münster PBF-File starten
+docker run -d \
+  --name ors-app \
+  -u $(id -u):$(id -g) \
+  -p 8080:8082 \
+  -v ./ors-docker-latest:/home/ors \
+  -e CONTAINER_LOG_LEVEL=INFO \
+  -e XMS=1g \
+  -e XMX=2g \
+  -e REBUILD_GRAPHS=true \
+  --env-file ors_car_muenster_latest.env \
+  openrouteservice/openrouteservice:latest
+
+# Neuen Container mit historischem Münster PBF-File starten
+docker run -d \
+  --name ors-app-historic \
+  -u $(id -u):$(id -g) \
+  -p 8081:8082 \
+  -v ./ors-docker-historic:/home/ors \
+  -e CONTAINER_LOG_LEVEL=INFO \
+  -e XMS=1g \
+  -e XMX=2g \
+  -e REBUILD_GRAPHS=true \
+  --env-file ors_car_historic_muenster.env \
+  openrouteservice/openrouteservice:latest
```

Comment:
Eventuell machts hier Sinn, mit `docker compose` zu arbeiten?
Die beiden `env`-files gibts ja eh schon, und hier wirds vllt bisschen übersichtlicher, das in einem `docker-compose.yml` statt in zwei `docker run` commands zu haben?

Dann hätte man das auch noch demonstriert, und wenn wir die passende `docker-compose.yml` hinterlegen ist die ja mit auch schnell ge-cURLed :)